portfolio2.css

```diff
@@ -335,22 +356,29 @@ fa {
    
    @media only screen and (max-width:800px){
 	   .navbar{
-		   align-items: flex-start;
+	
         flex-direction: column;
         box-sizing: border-box;
 		width: 100%;
+		border-radius: 100px;
            }
-		   .navbar ul{
```

Comment:
Very long media query, my guess is that many of the changes won't be necessary, but I really suck at CSS!